pkg/lwip/contrib/lwip.c

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
+ * @author Martine Lenders <mlenders@inf.fu-berlin.de>
+ */
+
+#include "lwip/tcpip.h"
+#include "lwip/netif/netdev2.h"
+#include "lwip/netif.h"
+
+#include "netdev2_tap.h"
```

Comment:
No, but until there is no GNRC-less auto-init there is no easy way around editing this file.